jdaviz/core/tests/test_config_detection.py

```diff
@@ -1,37 +1,21 @@
 import pytest
 from astropy.utils.data import download_file
-from jdaviz.core.data_formats import identify_helper
 
-# URIs to example JWST/HST files on MAST, and their
-# corresponding jdaviz helpers:
-example_uri_helper = [
-     ['mast:HST/product/id4301ouq_drz.fits', 'imviz'],
-     ['mast:HST/product/ldq601030_x1dsum.fits', 'specviz'],
-     ['mast:HST/product/o4xw01dkq_flt.fits', 'specviz2d'],
-     ['mast:JWST/product/jw02072-c1003_s00002_nirspec_clear-prism-s200a1_x1d.fits',
-      'specviz'],
-     ['mast:JWST/product/jw01324-o006_s00005_nirspec_f100lp-g140h_s2d.fits',
-      'specviz2d'],
-     ['mast:JWST/product/jw01345-o001_t021_nircam_clear-f200w_i2d.fits', 'imviz'],
-     ['mast:JWST/product/jw01373-o028_t001_nirspec_g395h-f290lp_s3d.fits',
-      'cubeviz'],
-     ['mast:JWST/product/jw01373-o031_t007_miri_ch1-shortmediumlong_s3d.fits',
-      'cubeviz'],
-     ['mast:JWST/product/jw01783-o004_t008_nircam_clear-f444w_i2d.fits', 'imviz'],
-     ['mast:JWST/product/jw02732-o004_t004_miri_ch1-shortmediumlong_x1d.fits',
-      'specviz']
-]
+from jdaviz.core.data_formats import identify_helper
 
 
-@pytest.mark.skip(reason="filenames changed")
+# URIs to example JWST/HST files on MAST, and their corresponding jdaviz helpers.
 @pytest.mark.remote_data
 @pytest.mark.filterwarnings(r"ignore::astropy.wcs.wcs.FITSFixedWarning")
-@pytest.mark.parametrize(
-    "uri, expected_helper", example_uri_helper
-)
+@pytest.mark.parametrize(("uri, expected_helper"), [
+    ('mast:HST/product/jclj01010_drz.fits', 'imviz'),
+    ('mast:JWST/product/jw02732-o004_t004_miri_ch1-shortmediumlong_x1d.fits', 'specviz'),
+    ('mast:jwst/product/jw01538-o161_s000000001_nirspec_f290lp-g395h-s1600a1_s2d.fits', 'specviz2d'),
+    ('mast:JWST/product/jw02727-o002_t062_nircam_clear-f277w_i2d.fits', 'imviz'),
+    ('mast:JWST/product/jw02732-o004_t004_miri_ch1-shortmediumlong_s3d.fits', 'cubeviz')])
 def test_auto_config_detection(uri, expected_helper):
     url = f'https://mast.stsci.edu/api/v0.1/Download/file/?uri={uri}'
     fn = download_file(url, timeout=100)
     helper_name, hdul = identify_helper(fn)
     hdul.close()
-    assert helper_name == expected_helper
+    assert len(helper_name) == 1 and helper_name[0] == expected_helper
```

Comment:
Something must have changed since this test was written (but never run), helper_name is a list now. 🤷‍♀️

Reply:
So... looks like this behavior is also in v4.1.x. https://github.com/spacetelescope/jdaviz/pull/3443